src/qt/rpcconsole.cpp

```diff
@@ -335,6 +380,9 @@ void RPCConsole::setClientModel(ClientModel *model)
         ui->startupTime->setText(model->formatClientStartupTime());
 
         ui->networkName->setText(QString::fromStdString(Params().NetworkIDString()));
+
+        connect(model, SIGNAL(banListChanged()), this, SLOT(showOrHideBanTableIfRequired()));
```

Comment:
I'd rather group these near the banlist stuff or add some small comment, before this is getting confusing here?